### ISSUE TITLE
community info not correct during registering phase

### ISSUE THREAD

**Issue by brenzi** [encointer/explorer#113]
![image](https://user-images.githubusercontent.com/10366175/203852145-8bc3b421-371d-4056-9b86-33af25cfc33e.png)

issues:
* [x] MEETUPS ASSIGNED IN LAST CEREMONY: 0 is wrong
* [x] PARTICIPANTS REGISTERED IN LAST CEREMONY: 0 is wrong
* [x] PARTICIPANTS REGISTERED: 0 is wrong (should be the sum of all categories displayed above)
* [x] MEETUPS ASSIGNED: 0 : it should not say 0. either hide this field or say "awaiting..."....
* [x] tentative community growth could be calculated and should be, even if we don't check the rules for assignment

**Comment by Jay-Forrer** [comment]
I found that MEETUPS ASSIGNED IN LAST CEREMONY, PARTICIPANTS REGISTERED IN LAST CEREMONY, PARTICIPANTS REGISTERED and MEETUPS ASSIGNED get their data from the Map file. In the Map File what I've discovered is that Map File, the varbiables that should store the data stores that it is 0. Here an example with state participants. 
![image](https://user-images.githubusercontent.com/55161261/203990086-e8e891d7-f1e3-4b51-a3c8-e664be61ef35.png)
Maybe it is only a problem that we have in the registering phase? I'll make a fix in the Mapsidebar phase where If these variables are 0 I'll run a function that gets the real number
